src/components/ProfilePage/LeaderboardCard/LeaderboardCard.tsx

```diff
@@ -0,0 +1,64 @@
+import Link from 'next/link';
+import { useTranslation } from 'react-i18next';
+import { LEADERBOARD_LENGTH } from 'src/components/Leaderboard/Leaderboard';
+import type { LeaderboardEntryData } from '../../../hooks/useLeaderboard';
+import { useLeaderboardUser } from '../../../hooks/useLeaderboard';
+import { IconHeader } from '../Common/IconHeader';
+import {
+  CardButton,
+  CardButtonContainer,
+  LeaderboardUserPositionButton,
+  LeaderboardUserTitle,
+  RankContainer,
+  RankContentContainer,
+} from './LeaderboardCard.style';
+import { RANKIcon } from 'src/components/illustrations/IconRANK';
+import { Box } from '@mui/material';
+
+export const LeaderboardCard = ({ address }: { address?: string }) => {
+  const { data: leaderboardUserData }: { data: LeaderboardEntryData } =
+    useLeaderboardUser(address);
+  const { t } = useTranslation();
+  const userPage = Math.ceil(
+    parseFloat(leaderboardUserData?.position) / LEADERBOARD_LENGTH,
+  );
+  const position = leaderboardUserData?.position;
+
+  return (
+    <RankContainer>
+      <Box sx={{ width: '104px', height: 'auto' }}>
+        <IconHeader
+          icon={<RANKIcon size={20} />}
```

Comment:
Should not be full uppercase to stay consistent with the rest of the icon naming

Reply:
done